package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ferry-tempo-server",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Since this is a bug fix, the version bump is a patch (instead of a minor)
```suggestion
  "version": "1.4.1",
```

---

Fixed that. Note you may see changes in the other pull request. These were done assuming that change would come sequentially after this one to get the version numbers correct.